CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- changed: Update getInOutTokenAddresses to use EdgeTokens
```

Comment:
I don't think we need changelog entries for every commit, especially with weird internal details. From a user-facing standpoint, what changed? I would expect:

- fixed: Block VELO tokens from currency-code based swaps, since there are two tokens with the same name.
- fixed: Lock spookySwap and tombSwap to only work on the Fantom chain.

Reply:
The change is these evm swap providers now work with any token regardless of currency code. This is the actual fix for the task since VELOv1 would not be tradeable at all

Reply:
Ah, so say that in the release notes instead?